LeagueToolkit/IO/OBJ/OBJFile.cs

```diff
@@ -69,27 +173,40 @@ public void Write(string fileLocation)
 
         public void Write(Stream stream, bool leaveOpen = false)
         {
-            using (StreamWriter sw = new StreamWriter(stream, Encoding.UTF8, 1024, leaveOpen))
+            using (StreamWriter sw = new StreamWriter(stream, Encoding.ASCII, 1024, leaveOpen))
             {
                 foreach (string comment in this.Comments)
                 {
                     sw.WriteLine("#" + comment);
                 }
+                if (!string.IsNullOrEmpty(this.MaterialsFile))
+                {
+                    sw.WriteLine("mtllib " + this.MaterialsFile);
+                }
                 foreach (Vector3 vertex in this.Vertices)
                 {
                     sw.WriteLine(string.Format("v {0} {1} {2}", vertex.X, vertex.Y, vertex.Z));
                 }
                 foreach (Vector2 uv in this.UVs)
                 {
-                    sw.WriteLine(string.Format("vt {0} {1}", uv.X, 1 - uv.Y));
+                    string format = string.Format("vt {0} {1}", uv.X, 1 - uv.Y);
```

Comment:
Looking at it, these implicit `ToString`s on the floats should most likely be using a `NumberFormatInfo.InvariantInfo` or the likes argument, like in the `Read` code below.